cns/middlewares/k8sSwiftV2_linux.go

```diff
@@ -0,0 +1,124 @@
+//go:build linux
```

Comment:
This is redundant since the file ends with `*_linux.go`

Reply:
Removed.